lib/WorkflowMain.groovy

```diff
@@ -2,6 +2,8 @@
 // This file holds several functions specific to the main.nf workflow in the nf-core/sarek pipeline
 //
 
+import nextflow.Nextflow
```

Comment:
Do we need to import nextflow.Nextflow if we're not exiting?

Reply:
True. This can be removed.